### DISCUSSION TITLE
kyverno #6287 repeated `TLS handshake error` resolved by removing and resyncing kyverno

### DISCUSSION THREAD

**Discussion by cameronraysmith** [deployKF/deployKF#184] [Questions & Answers] [answered]
A few weeks after upgrading `0.1.2` to `0.1.5` I ran into a problem that looked like https://github.com/kyverno/kyverno/issues/6287. 
Whatever the cause, it was able to be resolved by completely removing kyverno and resyncing.

**Comment by thesuperzapper** [comment, marked as answer]
@cameronraysmith there is a lot of discussion about Kyverno issues at the moment (https://github.com/deployKF/deployKF/issues/180, https://github.com/deployKF/deployKF/issues/183).

The current plan is to remove the dependency on Kyverno for cloning secrets (but leave it for the other use cases like triggering deployment restarts on secret change) so we can update to a much newer version of Kyverno.

The main reason we require exactly version `1.10.0` is related to it generate type rules, which are poorly implemented.

**Reply by cameronraysmith** [reply]
Thank you for the information and pointers.
